specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1785,22 +1785,22 @@ support deployment of Resource Adapters, as defined in the Connector
 specification, and must support the deployment of applications that use
 Resource Adapters.
 
-The Connector specification is available at
-_http://jcp.org/en/jsr/detail?id=322_ .
+The Jakarta EE Connectors specification is available at
+_https://jakarta.ee/specifications/connectors/1.7.2_ .
 
-=== Web Services for Jakarta EE 1.4 Requirements
+=== Jakarta EE XML Web Services 2.3 Requirements
```

Comment:
hmmm...  2.3?  Something is out of whack here...  We can't just change 1.4 to 2.3...  Maybe it meant "Web Services for Java EE 1.4" originally?  Instead of "Web Services 1.4 for Java EE"?

Reply:
I just removed the number entirely.

Reply:
Um, this looks wrong.  Those are two completely different specs.

Reply:
Thanks @bshannon... this change was reverted.